airbyte-integrations/connectors/source-alpha-vantage/source_alpha_vantage/alpha_vantage.yaml

```diff
@@ -0,0 +1,107 @@
+version: "0.1.0"
+
+definitions:
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_alpha_vantage/schemas/{{ options['name'] }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      class_name: source_alpha_vantage.object_dpath_extractor.ObjectDpathExtractor
+      field_pointer:
+        - "{{ options['dpath'] }}"
+      inject_key_as_field: "{{ options['key_field'] }}"
+  requester:
+    url_base: "https://www.alphavantage.co"
+    http_method: "GET"
+    authenticator:
+      type: NoAuth
+    request_options_provider:
+      request_parameters:
+        apikey: "{{ config['api_key'] }}"
+        symbol: "{{ config['symbol'] }}"
+        function: "{{ options['function'] }}"
+        datatype: json
+        outputsize: "{{ config['outputsize'] }}"
+        interval: "{{ config['interval'] }}"
+        adjusted: "{{ config['adjusted'] }}"
```

Comment:
Small detail @isaacharrisholt.

@girarda I saw one bug with this connector:
`https://www.alphavantage.co/query?apikey=****&symbol=IBM&datatype=json&outputsize=compact&interval=60min`
The request parameter `function` is not added to the url. If I change the parameter to a hardcoded string it works fine, example `TIME_SERIES_DAILY`. Any idea?

Reply:
@marcosmarxm I think this is an issue that was solved recently. You might need to reinstall the CDK with the newest version.

Reply:
I'll give a try, maybe I was using the wrong pyenv =p